tests/assets/smb_example_config.yml

```diff
@@ -0,0 +1,26 @@
+root-dir: ~/Documents/HSR/semester_06
+
+global-ignore:
+  - Thumbs.db
+  - .DS_Store
+
+plugins:
+  - name: skripte-server
+    type: smb
```

Comment:
This is okay-ish. But still, what is the "type" of a connection? If we call this thing "connection", we might as well just call this `plugin:` as the user actually defines what plugin to use. On the other hand, you could argue the plugin-thing is an implementation detail which shouldn't be exposed to the user, then `type:` is better.

Not sure, just some food for thought.

Reply:
I think plugin works fine.